db/migrate/20171031200739_drop_condition_modifier.rb

```diff
@@ -0,0 +1,30 @@
+class DropConditionModifier < ActiveRecord::Migration[5.0]
+  class Condition < ActiveRecord::Base
+    def reverse_expression(val, migration)
+      if val.include?("!ruby/object:MiqExpression")
+        val.sub!(/MiqExpression/, 'Hash')
+      elsif val.starts_with?('--- !')
```

Comment:
Should this just be `else`?

Reply:
If that's the case, maybe the condition should be reversed so that we return early